modules/programs/direnv.nix

```diff
@@ -118,73 +118,88 @@ in
     silent = mkEnableOption "silent mode, that is, disabling direnv logging";
   };
 
-  config = mkIf cfg.enable {
-    home.packages = [ cfg.package ];
+  config =
+    let
+      packageVersion = lib.getVersion cfg.package;
+      isVersion236rHigher = lib.versionAtLeast packageVersion "2.3.6";
```

Comment:
```suggestion
      isVersion236rHigher = lib.versionAtLeast packageVersion "2.36.0";
```

Reply:
oh good catch 😅

Reply:
Looks like this got reverted during the tweaking